buggy_utils.py

```
@@ -0,0 +1,107 @@
+
+# SQL Injection vulnerability
+def get_user(username):
+    import sqlite3
+    conn = sqlite3.connect("users.db")
+    query = "SELECT * FROM users WHERE name = '" + username + "'"
+    return conn.execute(query).fetchall()
+
+
+# Command injection vulnerability
+def run_diagnostic(host):
+    result = os.system("ping -c 1 " + host)
```

Ruff (0.15.12)

[error] Starting a process with a shell, possible injection detected

(S605)